### DISCUSSION TITLE
Velero  Backup Partially Failed  - AWS Volume Snapshot

### DISCUSSION THREAD

**Discussion by ShubhamGorlewar1** [vmware-tanzu/velero#6403] [Community support Q&A]
Aws takes little more time to create Snapshot of EBS Volumes
![test](https://github.com/vmware-tanzu/velero/assets/68949895/f181a6a2-0ad1-4376-996c-34e9ed1418bb)
 using velero. It  marks as partial failed and throw error region not found. 

ERROR : /persistentvolumes name: /pvc-a52525XXXXXXXXX11eaa5f error: /rpc error: code = Unknown desc = missing region in aws configuration

How we can fix this or is there any way  to make Velero wait for snapshot to complete

**Comment by ShubhamGorlewar1** [comment]
@sseago @ncdc @carlisia @dprotaso please help me out

**Comment by draghuram** [comment]
What version of Velero are you using? If it is 1.10 or later, you can set CSI snapshot timeout to a higher value. Default is 10 minutes. But you may still hit the issue fixed in the PR https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/175 but it is worth trying.

**Comment by ShubhamGorlewar1** [comment]
@draghuram  Velero  v1.11.0  I am using. The flag didn't worked  the same issue occurred again

**Comment by draghuram** [comment]
Will you be able to upload backup logs? That will make it easier to see what's going on.
